src/woo_publications/publications/api/serializers.py

```diff
@@ -185,7 +187,11 @@ class Meta:  # pyright: ignore
             "publicatiestatus": {
                 "help_text": _(
                     "\n**Disclaimer**: you can't create a {revoked} publication."
-                ).format(revoked=PublicationStatusOptions.revoked.label.lower())
+                    "\n\n**Disclaimer**: when you {revoked} a publication, the attached {published} documents also get's {revoked}."
+                ).format(
+                    revoked=PublicationStatusOptions.revoked.label.lower(),
+                    published=PublicationStatusOptions.published.label.lower(),
+                )
```

Comment:
This leads to weird sentences in the documentation, keep it simple:

```suggestion
                    "\n\n**Disclaimer**: when you revoke a publication, the attached published documents also get revoked."
                ).format(
                    revoked=PublicationStatusOptions.revoked.label.lower(),
                )
```